### PR TITLE
Add Codeception 3.0 compatibility

### PR DESCRIPTION
**Pull request by guillaume-ro-fr**
- Add Codeception 3.0 compatibility
- Fix MailCatcher API endpoint for message source

I can split the merge request if needed 😉

### CODE REVIEW

**Comment by serdyuka** [comment]
👍 
@captbaritone We really need a support for codeception v3. Thank you.

**Comment by dzoeteman** [comment]
Apologies for the wait, I was on holiday. I'll try and get this through today.

**Comment by JRCrawford** [comment]
Hi @captbaritone and @dzoeteman Is it possible if we can please get this PR pulled in? If there is however something that needs to be done please let me know and I'll take a look into it. 

(The issue I've run into is that I need to update Twig to use the BeelabRecaptcha2Bundle however this breaks my tests unless Codeception 3.0 is used)

**Comment by dzoeteman** [comment]
Sorry - took a little longer to get to do the review than I wanted to. 

@guillaume-ro-fr Thanks for your work! Your changes look great! 

I do notice PHPUnit 8 is spitting out warnings related to `assert(Not)Contains` usage:

```
[PHPUnit\Framework\Warning] Using assertContains() with string haystacks is deprecated and will not be supported in PHPUnit 9. Refactor your test to use assertStringContainsString() or assertStringContainsStringIgnoringCase() instead.
```

Could you change the `(dont)seeInEmailSubject`  & `(dont)seeInEmail` methods to use the `assertString(Not)ContainsString` methods?
I'd like to make sure we're not getting warnings - and it'd be good to have it ready for PHPUnit 9.


Thanks!

**Comment by guillaume-ro-fr** [comment]
@dzoeteman Thanks for your review ! The fix is pushed

**Comment by dzoeteman** [comment]
@guillaume-ro-fr Looks great, thanks for the quick fix!

**Comment by dzoeteman** [comment]
New version is released :) (2.1.0)

File: src/Module/MailCatcher.php [hunk 1]
```diff
@@ -365,7 +366,7 @@ protected function emailFromId($id)
      */
     protected function seeInEmailSubject(Email $email, $expected)
     {
-        $this->assertContains($expected, $email->getSubject(), "Email Subject Contains");
+        $this->assertStringContainsString($expected, $email->getSubject(), "Email Subject Contains");
```
Comment (Jamesking56) [inline src/Module/MailCatcher.php]: This change breaks backwards compatibility with Codeception 2. `assertStringContainsString` was added in v3.

Either update `composer.json` to allow 2.1 of this package to only require v3 or update this method to be backwards compatible.

**Pull request merged**